ckanext/recombinant/tests/test_canonicalize.py

```diff
@@ -137,14 +137,16 @@ def test_timestamp():
 
 def test_money():
     dt = 'money'
-    assert canonicalize(2019, dt, False) == '2019'
-    assert canonicalize(42.0, dt, False) == '42.0'
+    assert canonicalize(2019, dt, False) == '2019.00'
+    assert canonicalize(42.0, dt, False) == '42.00'
```

Comment:
@RabiaSajjad the excel template and the webform does not enforce decimals. So the tests should contain a mix of single decimals, no decimals, and more than two decimals.

Reply:
@RabiaSajjad sorry I am stupid. this does that HAHAHA